frontend/package.json

```diff
@@ -29,10 +32,17 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "sirv": "^2.0.3",
+    "styled-components": "^6.1.8",
     "ts-node": "^10.9.1",
     "typescript": "^5.3.3",
     "vike": "^0.4.156",
     "vite": "^5.0.10"
   },
-  "type": "module"
+  "type": "module",
+  "devDependencies": {
```

Comment:
Just wanted to nudge this comment. A couple of options as I see them:

1. Move these `devDependencies` into `dependencies` and remove `devDependencies`
2. Limit `dependencies` to just the following (please double check I have these right :joy: ), and move the rest into `devDependencies`:
    * `@react-oauth/google`
    * `styled-components`
    * `react`
    * `react-dom`

Reply:
:thinking: -- No. TBH I think it can actually probably be removed as the corresponding dev server was ripped out in [this commit](https://github.com/openstax/rope/pull/1/commits/ff02dce6cc89d744273bb3f4d67549615e6e287a) and we now use `vite`'s dev server. In production we use prerendered static files + NGINX.